src/Storefront/Resources/views/storefront/section/cms-section-block-container.html.twig

```diff
@@ -10,11 +10,10 @@
     {% endif %}
 
     {% set blockBgColor = block.backgroundColor %}
-    {% set blockBgImg = block.backgroundMedia|sw_encode_media_url %}
```

Comment:
Would this be a breaking change?

Reply:
This would be, but I think the switch from using a background image to a img element is a breaking change too.

Reply:
It will be released in the next major therefore it is okay :)